packages/cloud_security_posture/manifest.yml

```
@@ -108,5 +108,24 @@ policy_templates:
       - type: cloudbeat/cis_azure
         title: Azure
         description: CIS Benchmark for Microsoft Azure Foundations
+  - name: vuln_mgmt
+    title: Vulnerability Management
+    description: Scan for cloud resources vulnerabilities
+    categories:
+      - Vulnerability Management
```

```suggestion
      - vulnerability_management
```
It looks like `Credential Management` is `credential_management` [here](https://github.com/elastic/integrations/blob/789938e1e5bdcc5ffc4b968db8eda12882bc25c5/packages/1password/manifest.yml#L10)

---

I think the [CI](https://fleet-ci.elastic.co/blue/organizations/jenkins/Ingest-manager%2Fintegrations/detail/PR-5266/11/pipeline) didn't like that in any case

---

Should work based on the [latest version](https://github.com/elastic/package-registry/releases/tag/v1.18.0) of the package-registry